src/pages/workspace/companyCards/addNew/SelectCountryStep.tsx

```diff
@@ -0,0 +1,157 @@
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import FormHelpMessage from '@components/FormHelpMessage';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import Text from '@components/Text';
+import useDebouncedState from '@hooks/useDebouncedState';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import useThemeStyles from '@hooks/useThemeStyles';
+import searchOptions from '@libs/searchOptions';
+import type {Option} from '@libs/searchOptions';
+import StringUtils from '@libs/StringUtils';
+import Navigation from '@navigation/Navigation';
+import type {PlatformStackRouteProp} from '@navigation/PlatformStackNavigation/types';
+import type {WorkspaceSplitNavigatorParamList} from '@navigation/types';
+import {clearAddNewCardFlow, setAddNewCompanyCardStepAndData} from '@userActions/CompanyCards';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+
+type CountryStepProps = {
+    policyID?: string;
+};
+
+function SelectCountryStep({policyID}: CountryStepProps) {
+    const {translate} = useLocalize();
+    const route = useRoute<PlatformStackRouteProp<WorkspaceSplitNavigatorParamList, typeof SCREENS.WORKSPACE.COMPANY_CARDS_ADD_NEW>>();
+    const styles = useThemeStyles();
+    const policy = usePolicy(policyID);
+    const [currencyList = {}] = useOnyx(ONYXKEYS.CURRENCY_LIST, {canBeMissing: true});
+    const [countryByIp] = useOnyx(ONYXKEYS.COUNTRY, {canBeMissing: false});
+
+    const [searchValue, debouncedSearchValue, setSearchValue] = useDebouncedState('');
+    const [currentCountry, setCurrentCountry] = useState<string | undefined>('');
+    const [addNewCard] = useOnyx(ONYXKEYS.ADD_NEW_COMPANY_CARD, {canBeMissing: true});
+    const [hasError, setHasError] = useState(false);
+    const isEur = policy?.outputCurrency === CONST.CURRENCY.EUR;
+
+    const submit = () => {
+        if (!currentCountry) {
+            setHasError(true);
+        } else {
+            if (addNewCard?.data.selectedCountry !== currentCountry) {
+                clearAddNewCardFlow();
+            }
+            setAddNewCompanyCardStepAndData({
+                step: CONST.COMPANY_CARDS.STEP.SELECT_FEED_TYPE,
+                data: {
+                    selectedCountry: currentCountry,
+                },
+                isEditing: false,
+            });
+        }
+    };
+
+    useEffect(() => {
+        if (addNewCard?.data?.selectedCountry) {
+            setCurrentCountry(addNewCard.data.selectedCountry);
+            return;
+        }
+        if (isEur) {
+            setCurrentCountry(countryByIp);
```

Comment:
Only set to countryByIp if it is a european country (in `selectedCurrency?.countries`)

Reply:
Otherwise, just don't preselect